add mining reading and update lightning questions (#85)
chaincodelabs/seminars: lightning-protocol-development/payment-channels-htlcs.md

1. Given the fact that Lightning is punishment-based and requires nodes to be online, could an attacker trick you into publishing old states and lose all your funds?
2. How are accidental breaches handled on Lightning \(since at the moment this is mostly what is seen in the the wild\)?

> 3. Do you consider considering Lightning as a proof-of-stake system?

Reviewer: ```suggestion
3. Should the Lightning Network be considered a proof-of-stake system?
```
